ament_cmake_gen_version_h/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(ament_cmake_gen_version_h)
+include(CTest)
+
+find_package(ament_cmake_core REQUIRED)
+
+ament_package(CONFIG_EXTRAS "ament_cmake_gen_version_h-extras.cmake")
+
+# Default to C11
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 11)
+endif()
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(BUILD_TESTING)
+  # Simulate pre-installed package
+  set(ament_cmake_gen_version_h_DIR ${CMAKE_SOURCE_DIR}/cmake)
+  include (cmake/ament_cmake_gen_version_h.cmake)
+  find_package(ament_cmake_gtest REQUIRED)
+
+  ament_cmake_gen_version_h(NO_INSTALL TRUE)
```

Comment:
@serge-nikulin hmm, options do not need a literal boolean argument -- their presence (or absence) is the boolean.